build.gradle

```diff
@@ -362,3 +332,13 @@ task ciBuild {
     description 'This task is run by Travis. It builds the service and runs its unit tests. In addition it builds the functionaltests.'
     dependsOn ':service:test', ':functionaltests:compileTestJava'
 }
+
+task clean {
+    description 'By default :clean is executed only for :service and :functionaltests, ' +
```

Comment:
i think because only these 2 sub-projects have output, thus `clean` is not applied to root project, but in the same time `:functionaltests` writes _git push_ results are aggregated in `/build` directory, which is not cleared during simple `clean` task.